2020-10-29 03:36:12.858 - [Info] MITM Version: 1.26
2020-10-29 03:36:12.865 - [Info] Auto Access Enabled: true
2020-10-29 03:36:12.865 - [Debug] [Init] Auto Access Barrier: true
2020-10-29 03:36:12.889 - [Info] SSH man-in-the-middle server for 172.20.0.203 listening on 0.0.0.0:10002
2020-10-29 03:54:37.419 - [Debug] [Connection] Attacker connected: 141.98.81.154 | Client Identification: SSH-2.0-Go
2020-10-29 03:54:38.001 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "none"
2020-10-29 03:54:38.002 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 03:54:38.219 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "password"
2020-10-29 03:54:38.221 - [Debug] [Auto Access] Attacker: 141.98.81.154, Threshold: 1, Attempts: 1
2020-10-29 03:54:38.221 - [Debug] [Auto Access] Compromising the honeypot
2020-10-29 03:54:38.221 - [Debug] [Auto Access] Adding the following credentials: "admin:admin"
2020-10-29 03:54:38.863 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-29 03:54:38.864 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.203
2020-10-29 03:54:38.974 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-29 03:54:39.409 - [Debug] [Connection] Attacker closed connection
2020-10-29 03:54:40.325 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-29 03:54:40.325 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-29 03:56:40.983 - [Debug] [Connection] Attacker connected: 113.172.176.45 | Client Identification: SSH-2.0-OpenSSH_7.9p1 Raspbian-10+deb10u2
2020-10-29 03:56:41.119 - [Debug] [Connection] Attacker connected: 113.172.176.45 | Client Identification: SSH-2.0-OpenSSH_7.9p1 Raspbian-10+deb10u2
2020-10-29 03:56:42.745 - [Debug] [Auth] Attacker 113.172.176.45 trying to authenticate with "none"
2020-10-29 03:56:42.745 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 03:56:42.769 - [Debug] [Auth] Attacker 113.172.176.45 trying to authenticate with "none"
2020-10-29 03:56:42.769 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 03:56:43.216 - [Debug] [Auth] Attacker 113.172.176.45 trying to authenticate with "password"
2020-10-29 03:56:43.217 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 03:56:43.217 - [Debug] [Auth] Attacker: 113.172.176.45 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 03:56:43.298 - [Debug] [Auth] Attacker 113.172.176.45 trying to authenticate with "password"
2020-10-29 03:56:43.298 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 03:56:43.299 - [Debug] [Auth] Attacker: 113.172.176.45 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 03:56:43.734 - [Debug] [Connection] Attacker closed the connection
2020-10-29 03:56:43.782 - [Debug] [Connection] Attacker closed the connection
